templates/web/fixmystreet.com/around/location_error.html

```diff
@@ -0,0 +1,31 @@
+[% IF location_error_pc_lookup %]
+
+    <div class="search-help">
+        <h2 class="search-help__header" role="alert">
+            [% location_error | safe %]
+        </h2>
+
+        <div class="search-help__tips">
+            <div class="search-help__tips__category">
+                <h3>[% loc('Searching by postcode?') %]</h3>
+                <ul>
+                    <li>[% loc('Check you <strong>haven’t swapped numbers and letters</strong>. <code>O</code>, <code>0</code>, <code>I</code> and <code>1</code> aren’t the same.') %]</li>
+                    <li><strong>[% loc('Don’t forget the space</strong> in your postcode.') %]</li>
```

Comment:
The translation should include the opening tag as well - e.g. if a translation needed to move the strong to the end of the phrase. But as this is .com you could not have any translation strings at all at present.

Reply:
Ooops, typo. Will fix.